summit.html

```diff
@@ -109,10 +68,418 @@ <h1>Agenda</h1>
+          <thead>
+            <tr>
+              <th>Topic</th>
+              <th>Level</th>
+              <th>Start time</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>
+                <details>
+                  <summary>Welcome, introductions &nbsp; agenda adjustments</summary>
+                  <div>
+                    <p>
+                      There will no doubt be some new faces so let's
+                      introduce ourselves before we start. Our hosts
+                      might also have practical matters to talk about.
+                    </p>
+                    <p>
+                      <b>Speaker:</b> Eiffel Technical Committee &amp; Erik Sternersson
+                    </p>
+                  </div>
+                </details>
+              </td>
+              <td><span class="tag bd-success">Beginner</span></td>
+              <td>9:00</td>
+            </tr>
+
+            <tr>
+              <td>
+                <details>
+                  <summary>Break</summary>
+                  <div>
+                    <p>
+                      Time to stretch our legs, recaffeinate our vessels and socialize
+                    </p>
+                    <p>
+                      <b>Speaker:</b> All
+                    </p>
+                  </div>
+                </details>
+              </td>
+              <td>
+              </td>
+              <td>10:00</td>
+            </tr>
+
+            <tr>
+              <td>
+                <details>
+                  <summary>Track 1: What is Eiffel?<br>&nbsp;&nbsp;&nbsp;Track 2: Stream Processing &amp; Event Aggregations</summary>
+                  <div>
+                    <p>
+                      We split in two tracks - one for beginners and one for more advanced Eiffel users
+                      <br>Track 1: An introduction to what Eiffel is and why you should care
+                      <br>Track 2: Presentation and discussion about event stream processing, aggregations and a proposal about a light-weight alternative to Eiffel Intelligence
+                    </p>
+                    <p>
+                      <b>Speakers:</b><br> Track 1: Erik Sternersson<br> Track 2: Emil Bäckmark &amp; Tobias Åkervik
+                    </p>
+                  </div>
+                </details>
+              </td>
+              <td><span class="tag bd-success">Beginner/Advanced</span></td>
+              <td>10:15</td>
+            </tr>
+
+            <tr>
+              <td>
+                <details>
+                  <summary>Eiffel Usage Stories</summary>
+                  <div>
+                    <p>
+                      How are we using Eiffel today in our various organizations?
+                      We hope to hear from you all if/how you are using today,
+                      what benefits you get from it and what pain points you
+                      experience.
```

Comment:
This... isn't exactly what I had mind. Which doesn't mean it's wrong. I was thinking something similar to the short talks we had in the Sep 2021 summit, with 10–15 minute talks showcasing particular things we've done with Eiffel. So more short demos (albeit possibly in slide presentation form) than "here's the 10,000 ft view of what company X is doing with Eiffel".

Reply:
Oh. Yes, you might be right. That's how we discussed it. With that we'd need to urge all participants to spend some more time on preps though. I'll rephrase this a bit to cater for that (as well maybe)